File: framework/cs/Devian.Core/src/LoadMode.cs
namespace Devian.Core
{
    /// <summary>
    /// Table loading mode.
    /// </summary>
    public enum LoadMode
    {
        /// <summary>
        /// 기존 캐시 유지 + 새 데이터 병합. key 충돌 시 overwrite.
        /// </summary>
        Merge,

        /// <summary>
        /// 기존 캐시 Clear 후 로드.
        /// </summary>
        Replace
    }
}

File: framework/ts/devian-core/src/LoadMode.ts
/**
 * Table loading mode.
 * @see Devian.Core.LoadMode (C#)
 */
export enum LoadMode {
    /**
     * 기존 캐시 유지 + 새 데이터 병합. key 충돌 시 overwrite.
     */
    Merge = 'merge',

    /**
     * 기존 캐시 Clear 후 로드.
     */
    Replace = 'replace',
}
